src/content/experimental/removing-pattern/removing.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Removing
+---
+
+### This action removes an object from a list or item; however, the object is not destroyed as a result of the action. Multiple objects can be removed at once.
+
+#### Status: 
+
+Experimental
+
+#### Maintainers:
+
+[Vikki Paterson](https://github.ibm.com/VICTORIA), [Arnaud Gillard](https://github.ibm.com/Arnaud-Gillard)
+
+<AnchorLinks>
+
+- [Low impact](#low-impact)
+- [High impact](#high-impact)
+
+</AnchorLinks>
+
+<ImageComponent cols="8" caption="Example of a remove pattern using a modal">
+
+![Example of a remove pattern using a modal in context](images/modal-in-context.png)
+
+</ImageComponent>
+
+
+## Low impact
+As removing does not destroy an asset, typically user confirmation is not required. 
+
+
+## High impact
+
+As removing does not destroy an asset, typically user confirmation is not required. 
```

Comment:
I think this was copied over on accident. Looks good to me once it's removed.

Reply:
Just made the changes